filesystem_service/tox.ini

```diff
@@ -22,26 +22,28 @@ envlist =
 basepython = python3.6
 deps =
     pytest>=3.1,<3.2
+    inotify
 usedevelop = True
-commands = {posargs:py.test -l -vvv '{toxinidir}/tests'}
+commands = {posargs:py.test -l -vvv '{toxinidir}/src/tests'}
 
 [testenv:flake8]
 deps =
     flake8>=3.3,<3.4
     flake8-import-order>=0.12,<0.13
     flake8-quotes>=0.11,<0.12
-commands = {posargs:flake8 '{toxinidir}/setup.py' '{toxinidir}/src' '{toxinidir}/tests'}
+commands = {posargs:flake8 '{toxinidir}/setup.py' '{toxinidir}/src' '{toxinidir}/src/tests'}
 
 [testenv:pylint]
 deps =
     pylint>=1.7,<1.8
+    inotify
```

Comment:
This is not supposed to be here, but in `requirements.in`/`requirements.txt`.

Reply:
moved to requirments.txt